Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/PatientDiscovery201306PolicyChecker.java

```diff
@@ -53,43 +53,21 @@ public static PatientDiscovery201306PolicyChecker getInstance() {
     }
 
     public boolean check201305Policy(PRPAIN201306UV02 message, II patIdOverride, AssertionType assertion) {
+        return check201305PolicyIncomingOutgoing(message, assertion);
```

Comment:
patIdOverride doesn't seem to be used here.  Is there any reason why it wasn't deleted?  Could then condense the two methods as they would then share the same input and output.

Reply:
Fixed.